tests/DecimalNumberTest.php

```diff
@@ -6,17 +6,17 @@
  * @license   https://opensource.org/licenses/MIT MIT License
  */
 
-namespace PrestaShop\Decimal\tests\Unit\Core\Decimal;
+namespace PrestaShop\Decimal\Test;
 
-use PrestaShop\Decimal\Number as DecimalNumber;
+use PrestaShop\Decimal\DecimalNumber;
 use PrestaShop\Decimal\Operation\Rounding;
 
-class NumberTest extends \PHPUnit_Framework_TestCase
+class DecimalNumberTest extends \PHPUnit_Framework_TestCase
 {
 
     /**
      * Given a valid number in a string
-     * When constructing a DecimalNumber with it
+     * When constructing a Number with it
```

Comment:
Why changing ?

Reply:
It's a merge request should have be done ealier in a previous PR